apollos-church-api/src/data/Vimeo.js

```diff
@@ -14,11 +14,21 @@ class dataSource extends RESTDataSource {
   };
 
   getHLSForVideo = async (id) => {
+    const { Cache } = this.context.dataSources;
     // captures either vimeo/123 or 123
     const matches = id.match(/\/?(\d+)$/);
     if (matches && matches[1]) {
+      const cachedVideo = await Cache.get({
+        key: ['vimeo', id],
+      });
+      if (cachedVideo) return cachedVideo;
+
       const video = JSON.parse(await this.get(`videos/${matches[1]}`));
```

Comment:
couldn't we just throw a `cache({ttl: ONE_DAY})` here instead?

Reply:
or actually `options.cacheOptions` since this isn't a Rock request

Reply:
I like the Redis cache better here since it persists across restarts and across servers, is more consistent, and is easy to flush.

Reply:
Gotcha. Not arguing but help me understand the difference, we have multiple server instances running? Currently it goes to LRU memory but could go to Redis if we turned that on right? But still wouldn't persist across multiple instances?

Reply:
> Currently it goes to LRU memory but could go to Redis if we turned that on right? But still wouldn't persist across multiple instances?

It actually would! Turning on the Redis cache would drop the value in Redis where either server could pick it up. And we would have increased visibility into the cache, and could potentially have an easier time of wiping it.

Reply:
so when we scale Heroku horizontally, that creates multiple server instances? Is that what they mean by that?

Reply:
Correct - horizontal == more dynamos == more instances of the server that don't share resources
vertical == bigger dynamos